### ISSUE TITLE
How about project sharing & managing platform among code for all groups

### ISSUE THREAD

**Issue by hoony-o-1** [Code-for-All/codeforall.org-concept-2017#2]
This platform that I want to suggest is being needed to us, Codenamu.
It's similar with [Code for Philly](http://codeforphilly.com), but it should be more independant to platform and can aggregate all of codeforall groups.

Functions(in my opinion)
- [x] list of projects of each codeforall group
- [x] list of updates of projects that are being runned or are done(of each codeforall group)
- [ ] list of people who are involved in each project

Wish
- [ ] list of people who are needed to the project
- [ ] join the project

**Comment by cdaloisio** [comment]
Sounds good! The Code for America API already provides some of these functions, we could just contribute to the API in order to create the functionality above instead of making a new set of data.
We could then use a frontend JS framework like EmberJS to create the site, simply calling the API for the data we need.

**Comment by hoony-o-1** [comment]
Great! I'd like to listen to other's opinions!

**Comment by ondrae** [comment]
There are a few versions of your ideas that I've seen used by the Brigades. The first like you mentioned is built by Code for Philly. It is called [Laddr](https://github.com/CfABrigadePhiladelphia/laddr). It manages projects, people, events, [all kinds of stuff](https://github.com/CfABrigadePhiladelphia/laddr#features). It even kicks information out to the CfAPI. It seems very useful for group management.

The second is a fork of Chicago's [projects](http://codeforsanfrancisco.org/beta/projects/) and [people](http://codeforsanfrancisco.org/beta/people/) page, that works by reading in the API. Its very useful to show off the activities of a group.

I'm arguing that [the purpose](https://github.com/Code-for-All/codeforall.org#goals) of the Code for All site is to show off our collected works and stories. The main audience would be funders and the media. For each individual group, we should link to their own websites. There they can use Laddr or the API to show off their great work in detail.

What do you think?

**Comment by pmackay** [comment]
@ondrae I'd agree with your purpose for the CfAll site. Simple, lightweight, points people to other places for more info.

**Comment by hoony-o-1** [comment]
Oh! Sure, I totally agree with your opinion that the CfAll site should be lightweight and has the main audience of funders and the media.

However, what I want to suggest is the site for sharing or collaborating projects among Code for All groups. The main audience becomes ourselves. If then, we can look into other groups' projects, be inspired by those projects, and even we can participate in those projects. This site could be different with CfA site which you're building, or maybe should be different :)

The first, Laddr which I've built for Codenamu is too much dependent to [emergence](http://emr.ge) platform. But the structure for project managing is really nice.

The second, Chicago's site is only for listing projects and people. We cannot check all updates of each project and who are involved in each project not developers using github, but designers or coordinators.

Is this site needless to us? or too complicate?

**Comment by KrzysztofMadejski** [comment]
You can always add designers and coordinators to github - shouldn't be a problem. 

As for site just for CfAll members: I don't see a need to create a specific site that's gonna have the same content as main CfAll site, but more detailed.. In that I agree with @ondrae.

For CodeForPoland we're building a wordpress site showcasing all of the brigades, projects and people. Hopefully we'll find time to add export to CfAll Api feature.

**Comment by cdaloisio** [comment]
I agree also that the project management at this stage should be managed by the individual 'Code for' sites. 

Based on the discussion so far, it seems that the first 2 functions at the top of this issue are relevant, which I have checked. 

Perhaps we can close this issue @thechunsik for the moment until we can confirm later that the goals of Code For All change?

I have created a site requirement list in #3 based on the [the goals](https://github.com/Code-for-All/codeforall.org#goals). Let's discuss and confirm the requirements.
